src/domain/entity/UniqueID.ts

```diff
@@ -0,0 +1,18 @@
+import * as uuid from 'uuid/v4';
+
+import { Identifier } from './Identifier';
+
+/* 
+ * Probably should be static method or something, need to rethink.
+ * It's clear that in this case static method does't fit the logic,
+ * but maybe some other approach exist.
+ * Also calling uuid isn't testable.
```

Comment:
https://jestjs.io/docs/en/es6-class-mocks

Reply:
Thanks! 
It seems like if I want to mock uuid I would mock 'require'. I suggest avoiding those kinds of mocks and rethinking the whole flow in the future.

Reply:
You could definitely do blackbox testing
- if uuid was passed it could be cheked for equality
- if uuid wasn't passed of was passed invalid uuid generated uuid could be checked for meeting uuid structure